### PR TITLE
Fixed deprecated warnings

### PR DESCRIPTION
**Pull request by Noam-Alum**
### Changes:

- **File Check Post-Render**: Removed `taxonomyterm` as noted in the [Hugo v0.116.0 release](https://github.com/gohugoio/hugo/releases/tag/v0.116.0).
- **Blog Count Pagination**: Replaced `paginate: 15` with the new `pagination` format:
	```yaml
	pagination:
	  pagerSize: 15
	```
- **Social Links**: Updated to the new syntax in `layouts/partials/common/opengraph.html`.
- **Page Navigation**: Updated `.Page.PrevPage` and `.Page.NextPage` to the new syntax in `layouts/blog/single.html`.

### Posable issues:

I can't explain why, but it seems like the nav menu items look different:

- Old:
    image
- New:
    image
    
Also if I try to use the website translated (Locally) I can't access the blog's area, but this seems to be not related to the changes as I tested this on a blank clone.

@codyro If you can generate a test website (from GitHub pages) like I've seen you do sometimes to verify that there is no drastic impact on the website that would be great :grin:. 

> https://github.com/AlmaLinux/almalinux.org/issues/631

### CODE REVIEW

**Comment by codyro** [comment]
https://9cd7587a.almalinux-org.pages.dev/

File: layouts/partials/common/opengraph.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
         {{- with $.Site.Params.images }}<meta property="og:image" content="{{ index . 0 | absURL }}"/>{{ end -}}
         {{- if not $.Site.Params.images -}}
             <!-- Default image if no other image is found -->
-            <meta property="og:image" content="{{ "images/og/newhero.png" | absURL }}"/>
+            <meta property="og:image" content="{{ "images/newhero.png" | absURL }}"/>
```
Comment (codyro) [inline layouts/partials/common/opengraph.html:18]: Why did you change this?

File: layouts/partials/common/opengraph.html [hunk 2]
```diff
@@ -47,14 +47,14 @@
 {{ end }}{{ end }}
 {{- end }}
 
-{{- /* Deprecate site.Social.facebook_admin in favor of site.Params.social.facebook_admin */}}
+{{- /* Deprecate site.Social.facebook_admin in favor of */}}
```
Comment (codyro) [inline layouts/partials/common/opengraph.html:50]: Comment disappeared

**Comment by codyro** [comment]
On my local browser(s) the NAV seems consistent. I see the difference in your screenshots, though 🤔 

Kapture 2024-09-14 at 11 33 57

File: layouts/partials/common/opengraph.html (hunk 1 above)
Comment (Noam-Alum) [reply layouts/partials/common/opengraph.html:18]: By mistake 🫠 I'll fix it.

File: layouts/partials/common/opengraph.html (hunk 2 above)
Comment (Noam-Alum) [reply layouts/partials/common/opengraph.html:50]: It's a comment regarding the issue we try to solve, we should remove the other one as well.

**Comment by Noam-Alum** [comment]
> https://9cd7587a.almalinux-org.pages.dev/

🙏

**Comment by Noam-Alum** [comment]
> On my local browser(s) the NAV seems consistent. I see the difference in your screenshots, though 🤔 
> 
> 
> 
> Kapture 2024-09-14 at 11 33 57
> 
> 
> 
> 

Probably nothing, I just wanted to make sure.

File: layouts/partials/common/opengraph.html (hunk 2 above)
Comment (codyro) [reply layouts/partials/common/opengraph.html:50]: Works for me. I just noticed it got cut off :).

Comment (bennyvasquez) [inline layouts/partials/common/opengraph.html:50]: @Noam-Alum should this comment be removed entirely, or should it be updated to reflect the new one? so

`{{- /* Deprecate site.Social.facebook_admin in favor of site.Params.social.facebook_admin */}}`

Comment (Noam-Alum) [reply layouts/partials/common/opengraph.html]: I think removing the comment would be the best choice as its not important to note a deprecated parameter. (Also anyone can see the file changes in GitHub if for reasons beyond me this would cause any harm in the future)

**Review by bennyvasquez — APPROVED** [review]
this fixes the errors not fixed in #654.

**Comment by Noam-Alum** [comment]
> this fixes the errors not fixed in #654.

Good! 😄
